package.json

```diff
@@ -4,9 +4,14 @@
   "scripts": {
     "postinstall": "curl -s https://raw.githubusercontent.com/reactorcore/pomander/master/bin/install | bash",
     "start": "nodemon server",
-    "test": "jest NODE_ENV=test grunt test",
+    "test": " mocha ./server --recursive  jest --testPathPattern ./client/test\/.js  NODE_ENV=test grunt test",
```

Comment:
Wait this is starting the mocha server, then running jest and then running grunt test? Why don't you do all of this in grunt? And simply run NODE_ENV=test grunt test